### PR TITLE
Add RSS feed

### LINKED ISSUE #107: Add RSS feed?
Suggested by @stevenjmesser.

### PR DESCRIPTION
**Pull request by frankieroberto**
Using instructions from [the Eleventy plugin documentation](https://www.11ty.dev/docs/plugins/rss/).

- [x] Link to the feed in `<head>`
- [x] Visible link?
- [x] Add author metadata?
- [x] Validate it

Fixes #107

### CODE REVIEW

**Comment by frankieroberto** [comment]
Adding feed to the html `<head>` could be enabled upstream by https://github.com/x-govuk/govuk-eleventy-plugin/pull/258

**Comment by paulrobertlloyd** [comment]
Wondering if feed template could be upstreamed to the plugin: https://github.com/x-govuk/govuk-eleventy-plugin/pull/258#issuecomment-2000256894

File: app/posts/index.njk [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+permalink: posts.xml
```
Comment (stevenjmesser) [inline app/posts/index.njk:2]:
```suggestion
permalink: feed.xml
```
I recommend using `feed.xml` or `atom.xml` as most RSS readers/browser extensions will check for that file if one isn't mentioned in the head.

**Review by stevenjmesser — APPROVED** [review]

**Comment by paulrobertlloyd** [comment]
Maybe we can add a link to the feed in the footer?

**Comment by frankieroberto** [comment]
@paulrobertlloyd @stevenjmesser added link to the feed to the footer.

I'll merge this so we can test it, and then we can upstream it to the plugin in [PR #258](https://github.com/x-govuk/govuk-eleventy-plugin/pull/258).

**Pull request merged**